kolibri/core/assets/src/core-base.vue

```diff
@@ -18,33 +12,21 @@
 
 </template>
```

Comment:
@radinamatic this template represents the base for a page -- the `main` and `kolibri-nav` tags are application-specific. Main will be filled in with content based on the page you're on (e.g. different for the learn page vs user management). The kolibri-nav template is found in `navigation.vue`.